lib/tasks/job_runs.rake

```diff
@@ -1,30 +1,32 @@
 namespace :job_runs do
   task run: :environment do
     warn 'Calling job runner. See rails log for output.'
+    @keep_jobs_loop = true
+    @jobs_pid_file = Rails.root.join('tmp', 'job_runs-0.pid')
 
-    keep_loop = true
-    puts "rake job_runs:run with PID #{Process.pid}" if keep_loop
+    puts "rake job_runs:run with PID #{Process.pid}" if @keep_jobs_loop
 
-    `echo "#{Process.pid}" > #{Rails.root}/tmp/job_runs-0.pid`
+    File.open(@jobs_pid_file, 'w') { |file| file.write(Process.pid) }
 
     def shut_down
-      `rm -rf #{Rails.root}/tmp/job_runs-0.pid`
-      keep_loop = false
-      puts "\nShutting down gracefully..." unless keep_loop
+      File.unlink(@jobs_pid_file)
```

Comment:
this can raise an exception if the file doesn't exist

Reply:
I think that is the point. We want to see this error if it occurs. The file should always exist.

Reply:
If two are run concurrently one can cleanup and the other will fail.

Reply:
We could kill all running processes matching 'job_runs:run'. However, I'm worried about unexpected behaviour if the concurrently running processes. It seems better to have processes fail and generate and error. 

See my next comment. I'm wondering if @brodygov has an opinion on this.